lib/workers/repository/error.js

```diff
@@ -1,13 +1,15 @@
-const { raiseConfigWarningIssue } = require('./error-config');
+import { logger, setMeta } from '../../logger';
+import { raiseConfigWarningIssue } from './error-config';
 
-module.exports = {
+export default {
```

Comment:
why not simply  `export default handleError`

then we can use it like: `const handleError = require('./error').default;`